myscript.js

```diff
@@ -0,0 +1,18 @@
+const hamburger = document.querySelector('.hamburger');
+const navbar = document.querySelector('.navbar');
+const closeBtn = document.querySelector('.close-btn');
+
+hamburger.addEventListener('click', () => {
+  hamburger.classList.toggle('active');
+  navbar.classList.toggle('active');
+});
+
+closeBtn.addEventListener('click', () => {
+  hamburger.classList.toggle('active');
+  navbar.classList.toggle('active');
+});
```

Comment:
- [ ] When I clicked on the hamburger and close button the mobile menu appears as show below on the video  .But as stated in the project requirement When the user clicks (or taps) the hamburger button, the mobile menu appears and When the user clicks (or taps) the close (X) button, the mobile menu disappear.So Kindly fix it 

Please check this [video](https://drive.google.com/file/d/1PNp4sFDHTbio3mkH86V_x2Y43AFMYNb3/view?usp=share_link) and take the appropriate adjustment .

Reply:
Thank you for the feedback. I have made the change to this now.